portfolio/src/main/webapp/script.js

```diff
@@ -19,10 +19,24 @@ function addRandomGreeting() {
   const greetings =
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
+//   Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
-  // Add it to the page.
+//   Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomFact() {
+    const facts = 
```

Comment:
consider having one fact per line which might be easier to read (see https://google.github.io/styleguide/jsguide.html#formatting-array-literals)

Reply:
Thanks, fixed this formatting in new commit.